Expensify/App
# Fix: back arrow for drawer navigation when browser is resized

### Details

https://github.com/Expensify/App/issues/7091#issuecomment-1059278952
### Fixed Issues

$ https://github.com/Expensify/App/issues/7091

### Tests |  QA Steps
1. Open the app on a web browser.
2. open any chat.
3. resize the browser to mobile view.
4. Click the back arrow to open the LHN.

Note: Test this PR with https://github.com/Expensify/App/pull/8067.


### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/156860526-4ed2df04-8ee5-4f6e-9697-13d94f4fba30.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/156914691-4934f665-d8ba-4b0f-a7fb-20ad15b16c6c.mp4


#### Desktop


https://user-images.githubusercontent.com/24370807/156914533-708da872-221e-4455-83e6-e134c1dec98b.mp4


#### iOS


https://user-images.githubusercontent.com/24370807/156914555-8ef95187-64ac-4810-a7a7-2a19f07b4d24.mp4


https://user-images.githubusercontent.com/24370807/155207955-cea8345f-c39b-4c3b-9c49-0e2f17c836fe.mp4


#### Android


https://user-images.githubusercontent.com/24370807/156912073-d3ca0791-e714-47c5-a3fa-e96bed283c33.mp4
